### ISSUE TITLE
Frame drop?

### ISSUE THREAD

**Issue by julienkosinski** [truedread/netflix-1080p#15]
Hello,

Thanks for the incredible pluggin! :)
I'm watching Lost In Space in Chrome in 1080p and I'm experiencing multiple visible dropped frames. Buffering is correct though.

I don't experience this kind of thing in Edge or Netflix Windows 10 app in 1080p.

Is there something fixable?

Thank you very much again for the wonderful stuff! :)

**Comment by julienkosinski** [comment]
I did not diabled Hardware Acceleration and this is enabled by default.
I remembered that I used deprecated tips for a problem I had (the --force-wave-audio on shortcut of Chrome). I deleted that and it seems better now. I'm gonna test a bit more before closing :).

**Comment by truedread** [comment]
Honestly there's no way around this. Netflix uses Widevine DRM on Chrome, and with the license policy they have set (SW_SECURE_DECODE), the Widevine library decodes the frames rather than the CPU/GPU decoding the frames. As such, the frames are decoded without any sort of hardware acceleration, meaning on lower end computers, there will be frame drops. This is actually the reason Netflix only lets ChromeOS play 1080p with Chrome, because ChromeOS has hardware Widevine DRM, meaning Widevine can actually decode the frames through hardware acceleration, and as such there would be no frame drops. The reason Edge / Windows 10 Netflix app doesn't drop frames is because those two use Playready for the streaming DRM, and since Playready is baked into Windows 10, I wouldn't be surprised if it found a way to take advantage of hardware accelerated decoding while still being secure.

TL;DR: Can't do anything about it. Blame DRM.

**Comment by tobeorla** [comment]
@truedread small question: How do i check on a provider which protection level/license policy has set?
I tried using `chrome://media-internals/` and checking the `mpd` manifest, but i only see `cenc` references, no SW_SECURE_CRYPTO , HW_SECURE_CRYPTO ,... : s

**Comment by nyuszika7h** [comment]
@natumbri @tobeorla Netflix actually enforces SW_SECURE_DECODE in its license policy. You can use this userscript to check: https://greasyfork.org/en/scripts/373903-eme-logger
